internal/controllers/nmc_reconciler.go

```diff
@@ -578,6 +590,8 @@ func (p *podManagerImpl) CreateLoaderPod(ctx context.Context, nmc client.Object,
 		}
 	}
 
+	labels.SetLabel(pod, actionLabelKey, WorkerActionLoad)
```

Comment:
actionLabelKey is already set in line 586

Reply:
Fixed, thanks!